src/geo/crs/CRS.Earth.distance.js

```diff
@@ -0,0 +1,18 @@
+export var wrapLng = [-180, 180];
+
+export var R = 6371000;
+
+/**
+ * @param {{ lat: number; lng: number; }} latlng1
```

Comment:
Docstrings not needed - doc shall be inherited from the `distance` method of the parent `L.CRS` class in the end.